### ISSUE TITLE
Split output into non-artifact vs SNV artifact, non-artifact vs indel artifact

### ISSUE THREAD

**Issue by davidbenjamin** [davidbenjamin/permutect#3]
That way, for example, if the model predicts a high probability of an indel artifact (eg near an STR) but a low probability of a SNV artifact it will not over-filter SNV calls.

Note that the appropriate structure for the output is NOT (true variant, SNV artifact, indel artifact).  Rather, it is two separate binary classifiers, and depending on the call we are filtering only one of the outputs is connected to the loss for any given datum.  That is, the model should output a logit for SNVs and a logit for indels (and maybe later we can split this further).  When filtering a SNV (indel) variant call, the loss is the binary cross entropy of the SNV (indel) logit.

The hardest part of this enhancement is that we will need to continue balancing our training data by both alt count and variant/artifact type.
